app/api/datastore/v1/reviewing.rb

```diff
@@ -32,14 +32,21 @@ class Reviewing < Base
         desc 'Mark an application as complete.'
         params do
           requires :application_id, type: String, desc: 'Crime Application UUID'
+          optional :decisions, type: [JSON] do
+            requires :interests_of_justice, type: JSON
+            requires :funding_decision, type: String
+            requires :comment, type: String
```

Comment:
Comment is optional. As will be means. Are you thinking of adding means later?

Reply:
I'll update to make the comment optional. Yes, we were initially thinking of leaving means for later, but I'm happy to add it now as well.

Reply:
Thanks. (Fine for means to be done later.)